pkg/cli/cli_test.go

```diff
@@ -1091,23 +1107,69 @@ func Example_user() {
 	// user ls --format=tsv
 	// 0 rows
 	// username
+	// user set FOO
+	// INSERT 1
+	// user set Foo
+	// INSERT 1
+	// user set fOo
+	// INSERT 1
+	// user set foO
+	// INSERT 1
 	// user set foo
 	// INSERT 1
+	// user set _foo
+	// INSERT 1
+	// user set f_oo
+	// INSERT 1
+	// user set foo_
+	// INSERT 1
+	// user set ,foo
+	// username ",foo" invalid; usernames are case insensitive, must start with a letter or underscore, may contain letters, digits or underscores, and must not exceed 63 characters
+	// user set f,oo
+	// username "f,oo" invalid; usernames are case insensitive, must start with a letter or underscore, may contain letters, digits or underscores, and must not exceed 63 characters
+	// user set foo,
+	// username "foo," invalid; usernames are case insensitive, must start with a letter or underscore, may contain letters, digits or underscores, and must not exceed 63 characters
+	// user set 0foo
+	// username "0foo" invalid; usernames are case insensitive, must start with a letter or underscore, may contain letters, digits or underscores, and must not exceed 63 characters
```

Comment:
sorry, my comment went to the main thread.
I was asking how to run commands on users that are also reserved keywords. Apparently, you can quote them.
This still raises some questions:
* postgres apparently enforces "Cannot be a word reserved by the database engine". Since we're going with their random syntax for usernames, should we follow this too?
* not having to quote most usernames but having to quote some reserved keywords feels like a good way to get into trouble when using automated tools.